example_programs/program_with_semantic_bug.p4

```diff
@@ -5,133 +5,260 @@ header ethernet_t {
     bit<16> eth_type;
```

Comment:
The old validation bug was not reproducible with the newest version of the compiler so I used a different one.